GTG/gtk/browser/taskbrowser.ui

```diff
@@ -438,7 +438,7 @@
                           </object>
                           <packing>
                             <property name="name">active_view</property>
-                            <property name="title">Active</property>
+                            <property name="title">Open</property>
```

Comment:
Aaah, the great debate of "Active" vs "Open" vs "To do" vs "All tasks" vs "Non-closed" tasks ;) I'm not sure which formulation is best.

Reply:
Been thinking about this:
- "All tasks" would also include closed tasks, which we don't show in that pane
- "Active" sounds like the tasks are being worked on or actionable right now
- "To Do" could work, but the second pane also includes to-dos